src/Turbo/composer.json

```diff
@@ -0,0 +1,82 @@
+{
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/azjezz/mercure-bundle"
```

Comment:
We should not forget to remove this once https://github.com/symfony/mercure-bundle/pull/42 is merged